File: 268-missing-number/missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(a) {
     let sumOfGivenArray = 0;
     let n = a.length;
    for(let i=0;i<n;i++) {
        sumOfGivenArray+= a[i];
    }
    let mySum = n*(n+1)/2;
    let res = mySum - sumOfGivenArray;
    return res;
    
};

File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {

        int len = nums.length;
        int summF = ((len*(len+1))/2);
        int tot = 0;

        for(int i=0;i<len;i++) {
             tot += nums[i];
        }

        int res = summF - tot;
        return res;






        
    }
}